websocket_back/src/room.js

```diff
@@ -64,12 +65,17 @@ export default class Room {
     console.log(`[SET_PLAYER_READY] Socket ${this.socket.id} is now ready`);
   }
 
-  setAllReady () {
+  _setAllReady () {
     this.store.clients = this.store.clients.map((client) => {
       client.isRrady = true;
```

Comment:
Typo here change : 
- `client.isRrady = true;`
to 
- `client.isReady = true;`